pm_yaml_config/pm_yaml_config.xml

```diff
@@ -0,0 +1,126 @@
+        <param type="text" name="struct_name" label="name" help="name to call the folder for containing each structure. This name will be postfixed with a unique number, e.g. struct_001."/>
+        <param type="select" name="charged" label="charged muon" value="true" help="if True, the muon will be considered charged instead of a muonium with an accompanying electron. Must be True for UEP calculations.">
+            <option value="true">true</option>
+            <option value="false">false</option>
+        </param>
+        <param type="integer" name="geom_steps" value="300" label="geometry optimization steps" help="maximum number of geometry optimisation steps."/>
+        <param type="float" name="vdw_scale" value="0.25" label="Van der Waal scale" help="Van der Waals scaling factor to use when generating muon sites to avoid existing atoms. Smaller values will allow muons to get closer to the other ions."/>
+        <param type="text" name="out_folder" value="muon-airss-out" label="output folder name" help="name to call the output folder used to store the input files that the script generates."/>
+        <param type="float" name="geom_force_tol" value="0.05" label="geometry optimization tolerance" help="tolerance on geometry optimisation in units of eV/Armstrong."/>
+        <conditional name="calculator_cond">
+            <param type="select" name="calculator" value="uep" help="calculator to generate structure files for. Must be a single word or a comma seperated list of values. Currently supported calculators are CASTEP, DFTB+ and UEP. Can also pass all as an option to generate files for all calculators.">
+                <option value="uep">uep</option>
+                <option value="castep">castep</option>
+                <option value="dftb+">dftb</option>
+                <option value="all">all</option>
+            </param>
+            <when value="uep">
+                <param type="float" name="uep_gw_factor" optional="true" value="4.0" label="gaussian witdh factor" help="Gaussian width factor for UEP calculation. Higher values will make the potential of atomic nuclei closer to the point-like approximation but may introduce artifacts."/>
+                <param type="text" name="uep_chden" optional="true" label="path to .den_fmt and castep files" help="Name of the .den_fmt file containing the electronic density for an Unperturbed Electrostatic Potential optimisation. The corresponding .castep file must be with the same seedname."/>
+            </when>
+            <when value="castep">
+                <param type="text" name="castep_command" optional="true" value="castep.serial" label="castep command" help="command to use to run CASTEP."/>
+                <param type="text" name="castep_param" optional="true" label="PARAM path" help="file path to the CASTEP parameter file."/>
```

Comment:
Mention that this is only required if downloading the file for command-line use on your own machine.

Reply:
castep is not supported by galaxy so it seems redundant?

Reply:
Yeah, you're right.